api/processors/actions/student/missed_class.py

```diff
@@ -0,0 +1,49 @@
+from datetime import timedelta
+
+from django.db import transaction
+
+from api.models import Group, Student
+from api.models.auxil.constants import STUDENT_CLASS_MISS_LIMIT
+from api.models.choices.log_event_type import StudentLogEventType
+from api.models.choices.status import StudentSituationalStatus
+from api.models.log_event import StudentLogEvent
+from api.processors.actions.student import StudentActionProcessor
+
+
+class StudentMissedClassProcessor(StudentActionProcessor):
+    def __init__(self, student: Student, group: Group, notified: bool):
+        self.notified = notified
+        self.group = group
+        super().__init__(student)
+
+    @transaction.atomic
+    def process(self) -> None:
+        self._create_log_events()
+        self._set_statuses()
+
+    def _is_student_reached_limit(self) -> bool:
+        last_two_weeks_date = self.timestamp - timedelta(weeks=2)
+        student_class_misses = StudentLogEvent.objects.filter(
```

Comment:
Do we really need this variable? Why we just don't get count here?

Also below it could be something like 
```
return student_class_misses_count >= STUDENT_CLASS_MISS_LIMIT
```

Reply:
Yes, it might make sense to use count instead of all if I got you correctly

Reply:
Yeah, and also return statement instead of 
```
if x: 
    return True
else:
    return False
``` 
it's easier to do just

```
return x
```

Reply:
Just don't get bogged down in code style issues @Roxe322 , remember that I suggested that I remain the only one (the youngest junior 😆 ) that gets reviewed on this level :)

Reply:
@Auch-Auch you can just merge whenever you feel you have addressed this comment